src/views/Project/index.jsx

```diff
@@ -356,23 +376,28 @@ export default class Project extends Component {
       [...issues, ...goodFirstBugs, ...mentoredBugs],
       'summary'
     );
+    const quickIntroduction = project.introduction.split(/(?=##\s)/)[0];
+    const fullIntroduction = project.introduction;
+    const source = this.state.open ? fullIntroduction : quickIntroduction;
+    const buttonText = this.state.open ? 'See Less' : 'See More';
 
     return (
       <Dashboard title={project.name}>
         {project.introduction && (
-          <ExpansionPanel>
-            <ExpansionPanelSummary expandIcon={<ChevronDownIcon />}>
-              <Typography variant="headline">Project Introduction</Typography>
-            </ExpansionPanelSummary>
-            <ExpansionPanelDetails>
-              <Typography variant="body1">
-                <Markdown
-                  source={project.introduction}
-                  renderers={{ link: this.linkRenderer }}
-                />
+          <Paper className={classes.projectSummary}>
```

Comment:
I think we can use the `Card` component instead and have the "See More" button be on the bottom left similar to https://material-ui.com/demos/cards/#simple-card.